modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java

```diff
@@ -92,13 +93,16 @@ protected void doTransactionalCommand(
 		String content = ParamUtil.getString(actionRequest, "content");
 		String description = ParamUtil.getString(actionRequest, "description");
 		String sourceURL = ParamUtil.getString(actionRequest, "sourceURL");
+		Date displayDate = ParamUtil.getDate(
+			actionRequest, "displayDate",
+			DateFormatFactoryUtil.getSimpleDateFormat("yyyy-MM-dd HH:mm"),
+			new Date());
```

Comment:
Is this last paran `new Date()`  necessary? 
The desciption of  `ParamUtil.getDate()` without that last parameter says:

> Returns the portlet request parameter value as a Date. **If the parameter is missing or not convertible to a Date, the current date is returned**